crates/sargon/src/wrapped_radix_engine_toolkit/low_level/execution_summary/execution_summary.rs

```diff
@@ -110,6 +110,37 @@ impl ExecutionSummary {
     }
 }
 
+impl ExecutionSummary {
+    pub fn classify_delete_accounts_if_present(&mut self) {
```

Comment:
this method is easier to test if you make one pure fuction, and then another which is `&mut self` which uses the pure func.

Reply:
This will be tested through manifest analysis, not meant to be tested in isolation.